### PR TITLE
Add CUDA 11.1 & 11.3 gpuci/rapidsai images

### PR DESCRIPTION
**Pull request by AyodeAwe**
This PR adds CUDA `11.1` & `11.3` `gpuci/rapidsai` images, enabling us to test previously unsupported versions of CUDA: `11.1` & `11.3` for CUDA-enhanced compatibility.

### CODE REVIEW

**Review by ajschmidt8 — COMMENTED** [review]
The `ci/axis/rapidsai-driver.yml` and `ci/axis/rapidsai-driver-arm64.yml` files have `DRIVER_VER` and `exclude` values that will also need to be updated.

**Review by ajschmidt8 — APPROVED** [review]
LGTM. Nice work. Should be all set to merge once CI passes :+1:

**Comment by AyodeAwe** [comment]
rerun tests

**Review by ajschmidt8 — APPROVED** [review]

**Review by ajschmidt8 — APPROVED** [review]

File: ci/axis/rapidsai-driver-arm64.yml [hunk 1]
```diff
@@ -16,7 +16,9 @@ RAPIDS_VER:
 CUDA_VER:
   - 11.5
   - 11.4
+  - 11.3
   - 11.2
+  - 11.1
```
Comment (ajschmidt8) [inline ci/axis/rapidsai-driver-arm64.yml]: If we're not publishing `11.1` and `11.0` for ARM, then all the `11.1` and `11.0` references in this file (and other ARM files) should be removed.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

File: ci/axis/miniforge-cuda-driver-arm64.yml [hunk 2]
```diff
@@ -33,28 +31,6 @@ DRIVER_VER:
...
-    DRIVER_VER: 460
-  - CUDA_VER: 11.1
-    DRIVER_VER: 465
-  - CUDA_VER: 11.1
-    DRIVER_VER: 470
-  - CUDA_VER: 11.1
-    DRIVER_VER: 495
-
   - CUDA_VER: 11.2
     DRIVER_VER: 450
```
Comment (ajschmidt8) [inline ci/axis/miniforge-cuda-driver-arm64.yml]: As mentioned on [this page](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html), driver versions `450` and `455` correspond to CUDA versions `11.0` and `11.1` respectively. Therefore, if we're not building `11.0` and `11.1` images for ARM, the respective driver versions should be removed throughout the ARM files as well.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**